docs/modules/ROOT/pages/general-info/technical-preview.adoc

```diff
@@ -1,11 +1,11 @@
 [[overview]]
 = Features in Technical Preview
```

Comment:
I thought that a Technical preview is discontinued also for Community. Does it still make sense to leave this section here? All new features are just released. When they are proven and don't change much, they are then considered for addition to the Enterprise edition.